orderbook/src/database/orders.rs

```diff
@@ -35,6 +36,11 @@ pub trait OrderStoring: Send + Sync {
     ) -> Result<Vec<Order>>;
 }
 
+pub struct SolvableOrders {
+    pub orders: Vec<Order>,
+    pub newest_settlement_block: u64,
+}
```

Comment:
Wouldn't it be equivalent to return `Vec<Orders>` as before along with a u64 block number from the trait method. This way, we wouldn't need the new "single purpose struct" for the change here? Of course, its also fine the way it is I am more curious about your choice here than suggesting a change.